source/help/messaging/sending-messages.rst

```diff
@@ -0,0 +1,60 @@
+.. _sending-messages:
+
+Messaging Basics
+================
+
+After a message has been sent, you or others can reply to it and link to it. Depending on which edition of Mattermost you have and how it's configured by the System Admin, you can also edit and delete your own messages.
+
+Message Types
+-------------
+
+Mattermost has two types of messages: posts and replies.
+
+Posts
```

Comment:
This is from the original doc. The terminology around posts, messages, and replies, should be examined at some point, but it's outside the scope of this PR.